scripts/CheckSdkSize.ps1

```diff
@@ -0,0 +1,56 @@
+﻿param (
+    [Parameter(Mandatory=$true)][string]$releasePath,
+    [Parameter(Mandatory=$true)][string]$ikey
+ )
+
+$ai = Join-Path $releasePath -ChildPath "ai.js";
+$aiMin = Join-Path $releasePath -ChildPath "ai.min.js";
+$aiMinZip = Join-Path $releasePath -ChildPath "ai.min.zip";
+
+# zip ai.min.js file
+# Compress-Archive -Path $aiMin -DestinationPath $aiMinZip;   #...I wish, but it doesn't work in VSTS ;(
+
+if(-not (test-path($aiMinZip))) {
+    set-content $aiMinZip ("PK" + [char]5 + [char]6 + ("$([char]0)" * 18))
+    (dir $aiMinZip).IsReadOnly = $false  
+}
+
+$shellApplication = new-object -com shell.application
+$zipPackage = $shellApplication.NameSpace($aiMinZip)
+$zipPackage.CopyHere($aiMin)
+Start-sleep -milliseconds 500
```

Comment:
To give some time to copy the file to a zip. Otherwise, it's showing that the zip files has only 24b.